src/nuklear.h

```diff
@@ -1256,10 +1256,10 @@ enum nk_panel_flags {
 /// __bounds__  | Initial position and window size. However if you do not define `NK_WINDOW_SCALABLE` or `NK_WINDOW_MOVABLE` you can set window position and size every frame
 /// __flags__   | Window flags defined in the nk_panel_flags section with a number of different window behaviors
 ///
-/// Returns `true(1)` if the window can be filled up with widgets from this point
-/// until `nk_end` or `false(0)` otherwise for example if minimized
+/// Returns a non-zero window pointer if the window can be filled up with widgets from this point
+/// until `nk_end` or `nullptr(0)` otherwise for example if minimized
 */
-NK_API int nk_begin(struct nk_context *ctx, const char *title, struct nk_rect bounds, nk_flags flags);
+NK_API struct nk_window* nk_begin(struct nk_context *ctx, const char *title, struct nk_rect bounds, nk_flags flags);
```

Comment:
Heh, this return type change is also a breaking change :)

Reply:
if(nk_begin(...)) is generally used and that will still behave the exact same way. Only if the user stores int it will break. Otherwise it will now provide functionality to immediately get the window you opened

Reply:
Agreed, this is not a problem for most users, but it is a breaking change for some. I'm only really pointing this out because the patch version was bumped, and not at least the minor version (maybe I could have been a little clearer, mb).

Reply:
Okay, will get on bumping the minor versions and merging asap